recipes_source/recipes/profiler.py

```diff
@@ -111,16 +107,17 @@
 # -------------------------  --------------  ----------  ------------  ---------
 
 ######################################################################
-# Here we see that, as expected, most of the time is spent in convolution (and specifically in ``mkldnn_convolution``
-# for PyTorch compiled with MKL-DNN support).
-# Note the difference between self cpu time and cpu time - operators can call other operators, self cpu time exludes time
-# spent in children operator calls, while total cpu time includes it.
+# 여기서 우리는 예상대로 대부분의 시간이 콘볼루션에서 보낸다는 것이 보입니다.
+# (특히 MKL-DNN 지원으로 컴파일된 Pytorch안에서의 ``mkldnn_convolution`` 기능)
+# 자체 CPU 시간과 CPU 시간 간의 차이점에 유의하셔야 합니다. 운용자들는 다른 운용자들을 호출을 할 수 있으며,
+# 자체 CPU 시간은 하위 운용자 호출에 소비되는 시간을 제외하며, 총 CPU 시간은 이를 포함합니다.
 #
-# To get a finer granularity of results and include operator input shapes, pass ``group_by_input_shape=True``:
+# 보다 세부적인 결과 정보를 얻고 운용자 입력 구조을 포함하려면 ``group_by_input_shape=True`` 를 패스합니다 :
 
-print(prof.key_averages(group_by_input_shape=True).table(sort_by="cpu_time_total", row_limit=10))
+print(prof.key_averages(group_by_input_shape=True).table(
+    sort_by="cpu_time_total", row_limit=10))
```

Comment:
```suggestion
print(prof.key_averages(group_by_input_shape=True).table(sort_by="cpu_time_total", row_limit=10))
```
두 개의 줄로 나누지 않아도 될 거 같아 제안드립니다.